options.html

```diff
@@ -31,13 +31,24 @@
                 <button id="save" data-message="optionsSave">Save</button>
             </div>
         </div>
-        <div class="row hidden" id="manualEntry">
-            <div class="col-xs-2" data-message="optionsShortCodeLabel">Short code:</div>
-            <div class="col-xs-8">
-                <input type="text" id="tenantCode" />
+        <div class="hidden" id="manualEntry">
+            <div class="row">
+                <div class="col-xs-2" data-message="optionsShortCodeLabel">Short code:</div>
```

Comment:
Can this just be a `<label>` rather than a div made to look like a label? I found [similar problems](https://github.com/talis/enterprise/issues/1757) with the other inputs here and we don't want to make things worse.

Reply:
I'd rather leave this as it is so that it's consistent with what's already there. Surely it's better to convert the divs to labels in a single pass as part of that ticket?

Reply:
> If we fix broken windows in functional-change pull-requests, we increase the size and complexity of our pull-request, as the functional changes cannot be easily disentangled from refactoring changes.

[Source](https://talis.github.io/topics/code-standards.html)

Whilst we could argue that this is adding additional broken windows, if we were to change this to `<label>` it would then be inconsistent with the rest of the repo. At which point we could then start saying this PR should fix them up as well.

I'm inclined to approve this PR because it follows existing patterns with the repo, and there is an [open issue](https://github.com/talis/enterprise/issues/1756) to look at the accessibility issue.